accelerator/roles/intel/tasks/verify_has_accelerators.yml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2024 Intel Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+- name: Initialise node accelerator status
+  ansible.builtin.set_fact:
+    node_has_accelerator: false
+
+- name: Check node accelerator status
+  ansible.builtin.shell: |
+    set -o pipefail
+    lspci | grep -i "{{ intel_gaudi_device_pattern }}"
```

Comment:
we dont need separate playbook just for this command

Reply:
We agreed to keep this for the moment